image-viewer.py

```diff
@@ -55,5 +58,82 @@ def render(self):
 
         self.root.bind("<KeyPress>", self.onKeyPress)
         self.root.mainloop()
+        
+    # Menu creation
+    def create_menu(self):
+        menubar = Menu(self.root) # Create the main Menu bar
+        self.root.config(menu=menubar) # Attach the Menu bar to the main root window
+        
+        file_menu = Menu(menubar, tearoff=False) # Create the 'File' sub menu
+        file_menu.add_command(label='Open Folder', command=self.open_folder, underline=0) # Open Folder
```

Comment:
Might be a job for another PR, depending on how big you think this would be, but perhaps there could be an option in the menu to Open File(s), so the user could navigate to a specific file via the file explorer and view it in the image viewer, or select multiple files in the file explorer and display in the same way as when you open a folder. That would save users having to create folders to put one or specific files in.

What are your thoughts?